src/Foundatio.AWS/Metrics/CloudWatchMetricsClientOptions.cs

```diff
@@ -1,14 +1,18 @@
 using System;
-using System.Collections.Generic;
-using Amazon;
-using Amazon.CloudWatch.Model;
-using Amazon.Runtime;
 
 namespace Foundatio.Metrics {
     public class CloudWatchMetricsClientOptions : MetricsClientOptionsBase {
-        public AWSCredentials Credentials { get; set; }
```

Comment:
I think you should be able to set the AccessKey, SecretKey, and Region manually on the options object in addition to being able to set the connection string.

Reply:
I think we should keep the same design pattern for AWS, Aliyun and Azure. The AccessKey, SecretKey, and Region are configured with ConnectionString in Aliyun and Azure, but only AWS is not.